src/rawSHA1_ng_fmt_plug.c

```diff
@@ -330,12 +330,10 @@ static void sha1_fmt_set_key(char *key, int index)
 	vtype  Z   = vsetzero();
 	vtype  X   = vloadu(key);
 	vtype  B;
-#if __AVX512F__ || __MIC__
-	uint32_t len = strlen(key);
-#else
-	uint32_t mask = vmovemask_epi8(vcmpeq_epi8(X, Z));
+
+	// FIXME: even uint64_t won't be long enough for AVX-1024
+	uint64_t mask = vcmpeq_epi8_mask(X, Z);
```

Comment:
Without OMP (non-OMP builds or formats or OMP_NUM_THREADS set to 1), `virtual` reports what you could expect as the `real` c/s rate if there were no other load, i.e., if your system has no other load, the reported `virtual` c/s rate should be exactly the same as the `real` c/s rate, or only slightly higher.
With OMP, a system without any other load should report `real` c/s rates of about `virtual * OMP_NUM_THREADS`.

May be we should print `1408 Kc/s` instead of `1408K c/s`, to make the difference more obvious, but that's for Solar to decide. (Such a change would require either `benchmark-unify` or `relbench` and possibly other tools parsing `john --test` output to be adjusted.)